README.md

```diff
@@ -24,9 +24,9 @@ Download Helm dependencies:
 
 ### Install Temporal
 
-Temporal can be configured to run with a couple of database choices.
+Temporal can be configured to run with a variety of different dependencies. By default this Helm chart deploys Cassandra, ElasticSearch, Kafka, Zookeeper, Promethueus and Grafana to support various features of Temporal. Cassandra can be swapped for MySQL if desired. The following sections will walk through deploying with everything included on a single node cluster and conclude with a configurtion that lets you bring all of your own dependencies.
 
-#### [Tiny] Batteries Included
+#### [Tiny] Batteries Included for Getting Started / Development
```

Comment:
What does [Tiny] mean?
I don't like "Batteries Included" because it begs too many questions. Try using a heading that doesn't use an analogy in this case.